src/sentry/incidents/subscription_processor.py

```diff
@@ -500,6 +511,15 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
 
         aggregation_value = self.get_aggregation_value(subscription_update)
 
+        self.has_feature_flag = features.has(
+            "organizations:anomaly-detection-alerts", self.subscription.project.organization
+        )
+
+        if self.has_feature_flag:
+            anomalies = self.get_anomaly_data_from_seer(aggregation_value)
```

Comment:
these are not necessarily 'anomalies' so maybe we should rename this?
`anomaly_check_results` or something? 🤔